esmvalcore/preprocessor/_time.py

```diff
@@ -1744,14 +1727,12 @@ def _check_cube_coords(cube):
     if not cube.coords('longitude'):
         raise CoordinateNotFoundError(
             f"Input cube {cube.summary(shorten=True)} needs a coordinate "
-            f"`longitude`"
-        )
+            f"`longitude`")
     lon_ndim = len(cube.coord_dims('longitude'))
     if lon_ndim != 1:
         raise CoordinateMultiDimError(
             f"Input cube {cube.summary(shorten=True)} needs a 1D coordinate "
-            f"`longitude`, got {lon_ndim:d}D"
-        )
+            f"`longitude`, got {lon_ndim:d}D")
```

Comment:
Codacy has rerun across the entire module, and, even if that's not code you introduced, it is code that your linter/styler has changed/reformatted (is it `black` or `precommit`?), so ideally, we shouldn't do this with every PR and be careful not to reformat the code to local standards